### PR TITLE
OS-253 : rename proposalId to pluginProposalId

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

Task: [OS-253](https://aragonassociation.atlassian.net/browse/OS-253)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

[OS-253]: https://aragonassociation.atlassian.net/browse/OS-253?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/subgraph/tests/admin/admin.test.ts [hunk 1]
```diff
@@ -63,7 +63,12 @@ test('Run Admin plugin (handleProposalCreated) mappings with mock event', () =>
   assert.fieldEquals('AdminProposal', entityID, 'id', entityID);
   assert.fieldEquals('AdminProposal', entityID, 'dao', DAO_ADDRESS);
   assert.fieldEquals('AdminProposal', entityID, 'plugin', pluginId);
-  assert.fieldEquals('AdminProposal', entityID, 'proposalId', PROPOSAL_ID);
+  assert.fieldEquals(
+    'AdminProposal',
+    entityID,
+    'pluginProposalId',
+    PROPOSAL_ID
```
Comment (Michael-A-Heuer) [inline packages/subgraph/tests/admin/admin.test.ts:70]: Should this constant be renamed as well?

File: packages/subgraph/CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed attribute of `proposalId` in all proposal entities to `pluginProposalId`.
```
Comment (brickpop) [inline packages/subgraph/CHANGELOG.md:12]: If it's a breaking change to external users, I would add some `BREAKING` mention

Comment (Michael-A-Heuer) [reply packages/subgraph/CHANGELOG.md:12]: Shouldn't this be reflected through the increment of the `MAJOR` version number according to the [semantic versioning conventions](https://semver.org/spec/v2.0.0.html)?

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 1. MAJOR version when you make incompatible API changes
> 2. MINOR version when you add functionality in a backwards compatible manner
> 3. PATCH version when you make backwards compatible bug fixes
>
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

File: packages/subgraph/tests/admin/admin.test.ts (hunk 1 above)
Comment (Rekard0) [reply packages/subgraph/tests/admin/admin.test.ts]: yes, thanks, amended

File: packages/subgraph/CHANGELOG.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Changed attribute of `proposalId` in all proposal entities to `pluginProposalId`.
+- Changed attribute of `proposalId` in all proposal entities to `pluginProposalId`. (this will break sdk/ui)
```
Comment (brickpop) [inline packages/subgraph/CHANGELOG.md:12]:
```suggestion
- BREAKING: Changed attribute of `proposalId` in all proposal entities to `pluginProposalId`
```

File: packages/subgraph/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/osx-subgraph",
-  "version": "1.0.1",
+  "version": "1.1.1",
```
Comment (brickpop) [inline packages/subgraph/package.json:3]: Why not 1.1.0?

**Review by brickpop — COMMENTED** [review]
Where is the change implemented?

File: packages/subgraph/package.json (hunk 4 above)
Comment (Rekard0) [reply packages/subgraph/package.json]: true, my bad

**Review by brickpop — DISMISSED** [review]

**Review by josemarinas — DISMISSED** [review]

**Review by josemarinas — APPROVED** [review]

**Review by brickpop — APPROVED** [review]

**Pull request merged**